PSScriptAnalyzerSettings.psd1

```diff
@@ -0,0 +1,7 @@
+@{
```

Comment:
Interesting! Would you document this to https://lombiq.atlassian.net/wiki/spaces/DEV/pages/43384906/PowerShell+and+Windows+Management+Framework, please?

Reply:
I've added it to the [PowerShell code conventions](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/43384918/PowerShell+code+conventions).

Reply:
Great! Let's also have https://lombiq.atlassian.net/browse/OSOE-150.